apps/dcos_dns/test/dcos_dns_mesos_tests.erl

```diff
@@ -53,6 +53,11 @@ hello_overlay_test_() ->
         {"hello-host-vip-0-server", fun hello_overlay_host_vip/0}
     ]}.
 
+dns_cache_test_() ->
```

Comment:
`dcos_dns_mesos_tests.erl` contains only tests for mesos state listener. Could you please move that new test to `dcos_dns_SUITE.erl`? Also I believe you even don't need lashup to cover this case.

Reply:
I kept it there because it required minimal changes.  I will try to move it to `dcos_dns_SUITE.erl`. lashup is just a wrapper around erldns in those test.